src/components/NotificationStack/index.js

```diff
@@ -13,16 +15,6 @@ class NotificationStack extends Component {
     notifications: []
```

Comment:
Why do we need to have this? @devanshu-santiment

Reply:
sorry, pushed a fix